dumper/lib-dumper-spi/src/main/java/com/google/edwmigration/dumper/plugin/lib/dumper/spi/SnowflakeLogsDumpFormat.java

```diff
@@ -38,4 +38,241 @@ enum Header {
     CreditsUsedCloudServices,
     QueryText
   }
+
+  interface WarehouseEventsHistoryFormat {
+
+    String AU_ZIP_ENTRY_NAME = "warehouse_events-au.csv";
```

Comment:
Initially I suggested it because I misunderstood the way time-series-ingestor scans for files; now I think we don't actually need a common prefix; we should just leave the original name unchanged. i.e. keeping "_history" and not adding "-au".